altair/utils/headless.py

```diff
@@ -100,66 +83,73 @@ def connection_ok():
             .catch(function(err) { console.error(err); });
         """}
 
-def save_spec(spec, fp, mode=None, format=None, driver_timeout=10):
-    """Save a spec to file
+def spec_to_image_mimebundle(spec, format, mode,
+                             vega_version,
+                             vegaembed_version,
+                             vegalite_version=None,
+                             driver_timeout=10):
+    """Conver a vega/vega-lite specification to a PNG/EPS image
```

Comment:
Is EPS supported?

Reply:
Mixing up my three letter acronyms... should be SVG